tools/install-deps.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh
+
```

Comment:
This file is generated, right? Maybe it makes sense to add a comment stating that e.g. `Do not edit manually; generated by $tool_name`.

It would also make sense to mention that this file is used by the CI.

I'm also wondering whether this could be generated on the fly. Generally I'd avoid having generated files within Git.

Reply:
An advantage is, that I can reuse the `install-deps.sh` script for the next task, installing refreshed dependencies from devel:languages:perl. I only need to pass a parameter from which repo to install, so I believe this will be the last file that I generate, making it 4 files in total (cpanfile, spec, Dockerfile, install-deps.sh).